llvm/utils/repo/Makefile

```diff
@@ -40,17 +49,18 @@ libunwind.a:
 		-D utils_dir=$(LLVM_REPO)/llvm/utils/repo                     \
 		-D CMAKE_INSTALL_PREFIX=$(LLVM)                               \
 		-D musl_install=$(MUSL)                                       \
-		../libunwind &&                                               \
-	make -j 8 &&                                                          \
+		../libunwind
+	make -j 8
 	make install
 
-libcompiler-rt.a:
-	if [ -d "$(LLVM_REPO)/build_libcompiler-rt" ];then                    \
-		rm -rf $(LLVM_REPO)/build_libcompiler-rt;                     \
-	fi
-	cd $(LLVM_REPO)  &&                                                   \
-	mkdir build_libcompiler-rt && cd build_libcompiler-rt  &&             \
-	export REPOFILE=$(LLVM_REPO)/build_libcompiler-rt/libcompiler-rt.db &&\
+.ONESHELL:
+libcompiler-rt.a $(CLANG_RT)/lib/linux/clang_rt.crtbegin-x86_64.t \
+	$(CLANG_RT)/lib/linux/clang_rt.crtend-x86_64.t \
+	$(LLVM_REPO)/build_libcompiler-rt/libcompiler-rt.db:

```

Comment:
You state the output directory for all of the targets except libcompiler-rt.a. What’s the reason? (Same comment applies to other targets, some of which have the output directory in the target name.)

Could the list of files be laid out so that it’s easier to read? I don’t think that the backslashes are necessary.

Reply:
> 
> 
> You state the output directory for all of the targets except libcompiler-rt.a. What’s the reason? (Same comment applies to other targets, some of which have the output directory in the target name.)
> 

Since libcompiler-rt.a isn't used by other target rules as `$<`, I did not give the output directory. To be consistent, I have added the output directory in commit https://github.com/SNSystems/llvm-project-prepo/commit/841f014334761d2b2bf926c7063a4085f53f9cf1.

>
> Could the list of files be laid out so that it’s easier to read? 
>

Fixed in the commit https://github.com/SNSystems/llvm-project-prepo/commit/841f014334761d2b2bf926c7063a4085f53f9cf1.

>
> I don’t think that the backslashes are necessary.
>

It is required since target and prerequisite need to be in the same line. If we remove it, we will get the following make error: `Makefile:37: *** missing separator.  Stop.`